python/cuml/common/memory_utils.py

```diff
@@ -238,13 +238,64 @@ def set_global_output_type(output_type):
 
     Parameters
     ----------
-    output_type : {'input', 'cudf', 'cupy', 'numba', numpy'} (default = 'input')  # noqa
+    output_type : {'input', 'cudf', 'cupy', 'numpy'} (default = 'input')
         Desired output type of results and attributes of the estimators.
+
         'input' will mean that the parameters and methods will mirror the
-        format of the data sent to the estimators/methods. 'cudf' will
+        format of the data sent to the estimators/methods as much as
+        possible. Specifically:
+
+            Input type -> Output type
+
+            cuDF DataFrame or Series -> cuDF DataFrame or Series
+
+            NumPy arrays -> NumPy arrays
+
+            Pandas DataFrame or Series -> NumPy arrays
+
+            Numba device arrays -> Numba device arrays
+
+            CuPy arrays -> CuPy arrays
+
+            Other __cuda_array_interface__ objects -> CuPy arrays
+
+        'cudf' will
```

Comment:
Weird line breaks?

Reply:
Needed so that these appear in different lines, otherwise they appear as a big blob of text
![image](https://user-images.githubusercontent.com/1116796/82930695-6a1efa00-9f4b-11ea-8370-a410dd3c97c2.png)